ibm/service/vpc/resource_ibm_is_bare_metal_server.go

```diff
@@ -569,6 +569,13 @@ func ResourceIBMIsBareMetalServer() *schema.Resource {
 										ValidateFunc:  validate.InvokeValidator("ibm_is_virtual_network_interface", "vni_name"),
 										Description:   "The name for this virtual network interface. The name is unique across all virtual network interfaces in the VPC.",
 									},
+									"protocol_state_filtering_mode": {
```

Comment:
missing test case, data source and data source test changes

Reply:
Test case added. Datasource changes not applicable as API response does not contain VNI details.